gitlab/objects.py

```diff
@@ -1249,6 +1249,18 @@ class ProjectKey(GitlabObject):
     requiredUrlAttrs = ['project_id']
     requiredCreateAttrs = ['title', 'key']
 
+    def enable(self, key_id, project_id):
```

Comment:
Done.

But I read from self.parent.id in ProjectKeyManaged. Check below...

(if you have another suggestion related to that, tell me)